views/partials/navbar.hbs

```diff
@@ -1,21 +1,45 @@
-<nav class="navbar navbar-expand-lg navbar-light bg-light" id="navbar">
-  <a class="navbar-brand" href="/">Keto Eating</a>
-
-  <form class="form-inline my-2 my-lg-0" action="/">
-    <input class="form-control mr-sm-2 search-bar" type="search" placeholder="Enter keywords (ex: pizza)" aria-label="Search" name="term">
-    <button class="btn btn-outline-success my-2 my-sm-0" type="submit">Search</button>
-  </form>
-
-  <ul class="nav navbar-nav navbar-right">
-    <div class="nav-links">
-      {{# if user}}
-        <li><a href="/dashboard" class="btn btn-success">Dashboard</a></li>
-	      <li><a href="/cart" class="btn btn-primary">Cart</a></li>
-        <li><a href="/logout" class="btn btn-link">Logout</a></li>
-      {{else}}
-        <li><a href="/signup" class="btn btn-success">Sign Up</a></li>
-        <li><a href="/login" class="btn btn-secondary">Login</a></li>
-      {{/if}}
-    </div>
-  </ul>
+<nav class="navbar navbar-expand-md navbar-light bg-light" id="navbar">
```

Comment:
can you fix tab indentation on this file? It seems to have 4 spaces instead of 2 (like the rest of the file)